src/containers/Register/index.jsx

```diff
@@ -1,25 +1,90 @@
-import * as React from "react";
-import Avatar from "@mui/material/Avatar";
-import Button from "@mui/material/Button";
-import CssBaseline from "@mui/material/CssBaseline";
-import TextField from "@mui/material/TextField";
-import Link from "@mui/material/Link";
-import Grid from "@mui/material/Grid";
-import Box from "@mui/material/Box";
-import LockOutlinedIcon from "@mui/icons-material/LockOutlined";
-import Typography from "@mui/material/Typography";
-import Copyright from "../../components/Copyright";
-import RegisterStyle from "./index.style";
-import backgroundImage from "../../assets/images/authbackground.jpg";
-import route from "../../constants/route";
+import React, { useState } from 'react';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import Avatar from '@mui/material/Avatar';
+import Button from '@mui/material/Button';
+import CssBaseline from '@mui/material/CssBaseline';
+import TextField from '@mui/material/TextField';
+import Link from '@mui/material/Link';
+import Grid from '@mui/material/Grid';
+import Box from '@mui/material/Box';
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+import Typography from '@mui/material/Typography';
+import { useHistory } from 'react-router-dom';
+import isEmail from 'validator/lib/isEmail';
+import Copyright from '../../components/Copyright';
+import RegisterStyle from './index.style';
+import backgroundImage from '../../assets/images/authbackground.jpg';
+import route from '../../constants/route';
+import api from '../../apis';
 
 const Register = () => {
-  const handleSubmit = (event) => {
+  const [checkEmail, setCheckEmail] = useState('');
+  const history = useHistory();
+
+  let resultCheckEmail = '';
+
+  if (checkEmail === '') {
+    resultCheckEmail = '';
+  } else if (!isEmail(checkEmail)) {
+    resultCheckEmail = 'Email is not valid';
+  }
+
```

Comment:
const renderCheckEmail = () =>{
if (!isEmail(checkEmail)) return  'Email is not valid';
return '';
}